README.md

```diff
@@ -86,9 +86,9 @@ The `<branded-checkout>` element is where the branded checkout Angular app will
   - `quarterly` - quarterly recurring gift
   - `annually` - annually recurring gift
 - `day` - for recurring gifts this defaults the gift's day of the month - *Optional* - can be `1` to `28`
-- `donor-details` - set default values for donor's name and contact info - *Optional* - can be provided as an inline string JSON string or as an Angular expression referencing a variable from an outside Angular module - should be in this format:
+- `donor-details` - name of default values object for donor's name and contact info - *Optional* - should be in this format:
```

Comment:
I think you could say something specific about `window` here. Maybe name of the window variable containing default values for... or something.

Same with line 62